api/src/middlewares/Authentication.ts

```diff
@@ -9,6 +9,8 @@ export const NO_AUTHORIZATION_ROUTES = [
 	"/ping",
 	"/gateway",
 	"/experiments",
+	"/store/skus/",
+	"/store/applications",
```

Comment:
No, we will be keeping the store auth-required, as smaller instances cannot cope with guest traffic in those routes.